boot/boot_test.go

```diff
@@ -86,6 +86,23 @@ func (s *baseBootenvSuite) mockCmdline(c *C, cmdline string) {
 	c.Assert(ioutil.WriteFile(s.cmdlineFile, []byte(cmdline), 0644), IsNil)
 }
 
+// mockAssetsCache mocks boot assets in cache, a list of (path, data) tuples,
+// where if data is omitted an empty files gets created.
+func mockAssetsCache(c *C, rootdir, bootloaderName string, cachedAssets [][]string) {
+	p := filepath.Join(dirs.SnapBootAssetsDirUnder(rootdir), bootloaderName)
+	err := os.MkdirAll(p, 0755)
+	c.Assert(err, IsNil)
+	for _, cachedAsset := range cachedAssets {
+		var content []byte
+		if len(cachedAsset) == 2 {
+			// we have a (path, data) tuple
+			content = []byte(cachedAsset[1])
```

Comment:
will we use this feature later, it doesn't seem used so far?

Reply:
Actually, this may have been a bit premature, I've dropped the code that used the data element, so we can use a simple []string in the function arguments now.